CHANGES.rst

```diff
@@ -1,30 +1,19 @@
 Changelog
 =========
 
-6.0.2 (unreleased)
+6.0.1 (unreleased)
 ------------------
 
 Minor changes:
 
-- ...
+- Test compatibility with Python 3.13
 
 Breaking changes:
 
 - ...
 
 New features:
 
-- ...
-
-Bug fixes:
```

Comment:
These lines need to  stay... That is where people fill things in until we release.